python/Makefile

```diff
@@ -1,3 +1,11 @@
+SHELL := /bin/bash
+
+ifndef SCALA_CLASS_PATH
+$(error SCALA_CLASS_PATH for python tests is not set. Please check out \
```

Comment:
IMHO `SCALA_CLASS_PATH` is a bit too generic name. I would rather use a more specific name, for example `MLEAP_RUNTIME_CLASSPATH`.